src/main/java/RequestVerifier.java

```diff
@@ -0,0 +1,67 @@
+package com.td.HTTPServer;
+
+public class RequestVerifier implements HTTPVerifierInterface {
```

Comment:
Right, this should be two separate classes. Parsing and verification are definitely two different responsibilities and hence this class as more than one reason it could be changed.